tools/scripts/ansible/upgrade-playbook.yml

```diff
@@ -1,7 +1,21 @@
 # A playbook for upgrading packages
 - hosts: all
+  vars_files:
+    - vars/main.yml
+    - vars/openliberty.yml
+    # - local-vars/local.yml
   become: yes
   tasks:
+  # TODO: should skip these
```

Comment:
I don't understand this "TODO" comment -- if this PR finishes the ticket, why a TODO? What is it that should be skipped?

Reply:
it should only restart them if they are running.

Reply:
looks like it's time for me to try this revised upgrade-playbook on smoketest

Reply:
I already tested it last time (April 25, 2022)! -- old ticket https://unicode-org.atlassian.net/browse/CLDR-15581